.github/workflows/ci-wordpress.yaml

```diff
@@ -0,0 +1,103 @@
+name: CI-Wordpress
+
+# ----------------------------------------------------------------------------------------------------------
+# CI Pipeline for deploy wordpress to shared hosting by ansistrano ansible role
+#
+# Original Author: Rostislav Udaltsov
+# Created: 09/19/2023
+# ----------------------------------------------------------------------------------------------------------
+
+on:
+  workflow_call:
+    inputs:
+      # Build configuration
+      host:
+        required: true
+        type: string
+        description: 'The address of the server'
+      username:
+        type: string
+        required: true
+        description: 'The linux OS username for connect to server via ssh'
+      environment:
+        required: true
+        type: string
+        description: 'The environment'
+      playbook_path:
+        required: true
+        type: string
+        description: 'TThe path to the file relative to the repository from which the deployment is made'
```

Comment:
pls install spellchecker. It would catch such issues.

Reply:
fix